src/assets/stylesheets/InternalStyles.scss

```diff
@@ -40,7 +40,6 @@
 @use "./SaveStatus" as *;
 @use "./ContextMenu" as *;
 @use "./FilePanel" as *; // needs to be below Button
-@use "./EmbeddedViewer" as *;
```

Comment:
🔧 annoyingly we're still using the embedded styles in the web component project 🤦 [here](https://github.com/RaspberryPiFoundation/editor-ui/blob/41ef481f63f732d13fa40fac76747d2695693d27/src/components/WebComponentProject/WebComponentProject.jsx#L5) 🤦

Reply:
Re-added